docs/source/en/installation.mdx

```diff
@@ -39,6 +39,11 @@ Activate the virtual environment:
 ```bash
 source .env/bin/activate
 ```
+Activate Virtual environment on Windows
```

Comment:
You should specify on the previous block (line 37) that it's for Linux and MacOs. All in all something like this:

````
Activate the virtual environment. On Linux and MacOs:
```bash
source .env/bin/activate
```
On Windows:
```bash
source .env/bin/activate
```
````